### PR TITLE
Align help text descriptions

### PR DESCRIPTION
**Pull request by dennisschagt**
Resolves https://github.com/newsboat/newsboat/issues/2705
Related https://github.com/newsboat/newsboat/issues/2546

Before/After:
image

### CODE REVIEW

File: newsboat.cpp [hunk 1]
```diff
@@ -131,15 +137,17 @@ void print_usage(const std::string& argv0, const ConfigPaths& configpaths)
...
-	std::cout << ss.str();
 
-	std::cout << '\n';
+		maxLength = std::max(maxLength, longcolumn.length());
+		helpLines.push_back({longcolumn, a.desc});
+	}
+	for (const auto& helpLine : helpLines) {
+		std::cout << std::string(8, ' ') << helpLine.first;
+		const auto padding = maxLength - helpLine.first.length();
+		std::cout << std::string(1 + padding, ' ') << helpLine.second;
```
Comment (Minoru) [inline newsboat.cpp:147]: This must be the ultimate nitpick, but I'd prefer to have two spaces between the option and its description -- this better separates the two.

```suggestion
		std::cout << std::string(2 + padding, ' ') << helpLine.second;
```

File: src/pbcontroller.cpp [hunk 2]
```diff
@@ -392,19 +394,24 @@ void PbController::print_usage(const char* argv0)
...
-		}
-		std::cout << a.desc << std::endl;
+
+		maxLength = std::max(maxLength, longcolumn.length());
+		helpLines.push_back({longcolumn, a.desc});
+	}
+	for (const auto& helpLine : helpLines) {
+		std::cout << std::string(8, ' ') << helpLine.first;
+		const auto padding = maxLength - helpLine.first.length();
+		std::cout << std::string(1 + padding, ' ') << helpLine.second;
```
Comment (Minoru) [inline src/pbcontroller.cpp:404]:
```suggestion
		std::cout << std::string(2 + padding, ' ') << helpLine.second;
```

File: src/pbcontroller.cpp [hunk 3]
```diff
@@ -392,19 +394,24 @@ void PbController::print_usage(const char* argv0)
...
-			"list https://groups.google.com/g/newsboat")
-		<< std::endl
-		<< _("For more information, check out https://newsboat.org/")
-		<< std::endl;
+	std::cout
+			<< _("Support at #newsboat at https://libera.chat or on our mailing "
+					"list https://groups.google.com/g/newsboat")
+				<< std::endl
+				<< _("For more information, check out https://newsboat.org/")
+				<< std::endl;
```
Comment (Minoru) [inline src/pbcontroller.cpp:414]: This looks uglier than it used to, can you please change it back?

**Review by Minoru — APPROVED** [review]
Thank you very much!

File: src/pbcontroller.cpp (hunk 3 above)
Comment (dennisschagt) [reply src/pbcontroller.cpp]: Indeed, looks like our formatter does not handle it well if the first string is not on the same line as the `std::cout`.
Fixed.

**Comment by Minoru** [comment]
Thank you very much!

**Pull request merged**